protostuff-runtime/test-map-collection-register.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mvn -o -DforkMode=never \
+    -Dprotostuff.runtime.pojo_schema_on_collection_fields=true \
```

Comment:
We should think about better way of doing such customizations. System properties is not the best option as it is not always available / working.